app/screens/channel/channel_base.js

```diff
@@ -134,6 +134,12 @@ export default class ChannelBase extends PureComponent {
         if (LocalConfig.EnableMobileClientUpgrade && !ClientUpgradeListener) {
             ClientUpgradeListener = require('app/components/client_upgrade_listener').default;
         }
+
+        if (this.props.currentTeamId &&
+            nextProps.channelsRequestFailed &&
```

Comment:
I rather not depend on this state as we are in progress of removing them, maybe have getChannels action emitting an event or select the right team instead

Reply:
You can even inspect the return value of the action and take another action from there actually

Reply:
This would go into 1.29 though right? If that is the case it should be added after the changes on perf that do not use `fetchChannelsAndMembers` if it is for a quick dot release then maybe, but should we have a dot release only for this that would be the question. @amyblais thoughts?

Reply:
Before I can answer we need to make sure there is a proper fix and figure out how to get the reporter to verify the fix.